[CI] `test_cli` failed on MacOS due to different `matplotlib` rounding precision

This last line: https://github.com/JaGeo/LobsterPy/blob/2ec4d19edab95a18c36817b5ab764de73d7330de/tests/cli/test_cli.py#L92

of https://github.com/JaGeo/LobsterPy/blob/2ec4d19edab95a18c36817b5ab764de73d7330de/tests/cli/test_cli.py#L72-L92

Would fail only on MacOS (passed on Ubuntu):
```
>                   assert plot_attributes[key] == pytest.approx(ref_value)
E                   assert [20.0, 12.36] == approx([20.0 ...96 ± 1.2e-05])
E                     
E                     comparison failed. Mismatched elements: 1 / 2:
E                     Max absolute difference: 0.0006797749978968426
E                     Max relative difference: 5.499797717611995e-05
E                     Index | Obtained | Expected                    
E                     1     | 12.36    | 12.360679774997896 ± 1.2e-05
```

--- a/tests/cli/test_cli.py
+++ b/tests/cli/test_cli.py
@@ -9,6 +9,7 @@ import sys
 from contextlib import redirect_stdout
 from pathlib import Path
 
+import matplotlib as mpl
 import matplotlib.pyplot as plt
 import matplotlib.style
 import numpy as np
@@ -71,6 +72,9 @@ class TestCLI:
 
     @pytest.mark.parametrize("args", test_cases)
     def test_cli_results(self, args, capsys, inject_mocks, clean_plot):
+        # Use Ubuntu default backend for consistency across OS
+        mpl.use("agg")
+
         test = get_parser().parse_args(args)
         run(test)
 

